tests/mailmap/blame.c

```diff
@@ -0,0 +1,70 @@
+#include "clar_libgit2.h"
+#include "git2/repository.h"
+#include "git2/blame.h"
+#include "git2/mailmap.h"
+#include "mailmap_helpers.h"
+
+static git_repository *g_repo;
+static git_blame *g_blame;
+static git_mailmap *g_mailmap;
+
+void test_mailmap_blame__initialize(void)
+{
+	g_repo = NULL;
+  g_blame = NULL;
```

Comment:
nit: here too, looks like this file is inconsistently formatted wrt tabs and spaces.